test/e2e/specs/Launch.spec.ts

```diff
@@ -73,9 +76,17 @@ const skip = true;
 
       const screenshotFile = path.join(screenshotsDir, `${global.lastTest.test.name.replace(/\s/g, '')}.png`);
       const imgBuffer = await browserWindow.capturePage();
-      fs.writeFileSync(screenshotFile, imgBuffer);
+      fs.writeFileSync(screenshotFile, imgBuffer.toBitmap());
     }
   });
 
   afterAll(async () => stopServe());
 });
+
+declare global {
```

Comment:
Workaround for `global`. Same as `shims.d.ts`. We need something better but for now, I did this.